### PR TITLE
Merkle Redeemer + Dripper Deployment

### PR DESCRIPTION
**Pull request by xklob**

### CODE REVIEW

File: proposals/dao/tip_121b.ts [hunk 1]
```diff
@@ -133,7 +140,7 @@ const validate: ValidateUpgradeFunc = async (addresses, oldContracts, contracts,
   // check initial balances of dripper & redeemer
   // ensure that initial balance of the dripper is a multiple of drip amount
   const fei = contracts.fei as Fei;
-  expect(await fei.balanceOf(rariMerkleRedeemer.address)).to.be.equal(rariMerkleRedeemerInitialBalance);
+  //expect(await fei.balanceOf(rariMerkleRedeemer.address)).to.be(rariMerkleRedeemerInitialBalance);
```
Comment (Joeysantoro) [inline proposals/dao/tip_121b.ts]: this should be uncommented

File: protocol-configuration/mainnetAddresses.ts [hunk 2]
```diff
@@ -1,6 +1,16 @@
...
+  rariMerkleRedeemer: {
+    artifactName: 'RariMerkleRedeemer',
+    address: '0xCAe4210e6676727EA4e0fD9BA5dFb95831356a16',
+    category: AddressCategory.TBD
+  },
+  merkleRedeemerDripper: {
+    artifactName: 'MerkleRedeemerDripper',
+    address: '0xF681F42f53D98A9136D090A04b47318C6961a832',
+    category: AddressCategory.TBD
+  },
```
Comment (Joeysantoro) [inline protocol-configuration/mainnetAddresses.ts:13]: checked deploy params

**Review by xklob — COMMENTED** [review]
LGTM.

File: proposals/dao/tip_121b.ts (hunk 1 above)
Comment (xklob) [reply proposals/dao/tip_121b.ts]: fixed

**Review by Joeysantoro — APPROVED** [review]

**Pull request merged**